### PR TITLE
chore: Set minimal supported Kubernetes version

### PR DESCRIPTION
**Pull request by MOZGIII**
Specify the minimal supported Kubernetes version, according to the #2222.

The file location is not set in stone, I'm open to changes.

I think this format is the best fit, since the supported versions aren't really a part of the installation domain.

The proposals at #2222 are conceptual and not concrete - that's by the design of the RFC process. We didn't complete the discussion at #2222, so we can finish it here.

### CODE REVIEW

**Comment by binarylogic** [comment]
Thanks.

> I think this format is the best fit, since the supported versions aren't really a part of the installation domain.

I think it is. Installation is all about supported platforms, etc. Is there a scenario where the version we'd communicate during installation would not align with our supported version?

**Comment by MOZGIII** [comment]
To me, the difference is the MSKV is our input parameters for the design, and our installation documentation is the output (i.e. the representation of the current state of things). I.e. we should derive the installation parameters from the supported versions spec.

So, I see those as different concerns. And one of the consequences for that would be that we can specify MSKV as `1.14`, but not specify it in the installation docs until we deliver. Similarly - if we change MSKV to `1.10` for example - we only change the version we specify at the installation docs once the code is implemented and the release supporting `1.10` is ready.

If you think it's unnecessary - we can only keep the MSKV at the installation meta. I don't mind, I just want to make sure that the idea behind my approach is communicated properly.

**Comment by binarylogic** [comment]
Yeah, that is the point of the `.meta/installation/*` files. Using the installation namespace isn't necessary and clearly just confuses things. Your file is fine, can you add it under a `supported_versions` table though?

**Comment by MOZGIII** [comment]
@binarylogic can you take over this PR and apply changes you see fit? :smile:
I don't think I understood what you mean (I see multiple competing interpretations), but it's probably quicker if you just do it.

File: .meta/supported_versions.toml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# This file holds generic specification of version constraints of the third-pary
```
Comment (MOZGIII) [inline .meta/supported_versions.toml:1]:
```suggestion
[supported_versions]
# This file holds generic specification of version constraints of the third-pary
```

@binarylogic like this?

**Pull request merged**